src/reducers/filter.js

```diff
@@ -0,0 +1,9 @@
+import {CHANGE_FILTER} from "../actions";
+
+export const filterReducer = (state = 'All', action) => {
```

Comment:
According to Airbnb [guideline](https://github.com/airbnb/javascript) it will be better if you use `dafault`.